### DISCUSSION TITLE
Flatpak versions doesn't recognize files

### DISCUSSION THREAD

**Discussion by RafaelLinux** [enzo1982/freac#407] [General Discussion]
I had installed Flatpak version (thru Discover - Plasma - in OpenSUSE). I tried, as always , dragging and dropping mp3 files to free:ac . However, it shows a lot of "unrecognized" format messages. I tried opening (importing) folder from "Files, ..." ... same result.

Finally, I downloaded AppImage and all works as it should. Again, a Flatpak version of an application is not working fine. So THANKS for giving us an AppImage option to dowload.

**Comment by zordaz** [comment]
This probably has to do with the security model of Flatpak. You could try installing Flatseal to see all the permissions and tinker a bit with the permissions of the Flatpak Fre:ac version.

**Reply by RafaelLinux** [reply]
Thank you. I tried Flatseal for another problematic Flatpak version of an application. But I gave up. I prefer to use AppImage format, that works in 99% cases while Flatpak about 50% and is a headache.
